### PR TITLE
LRQA-72706 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LRQA-72706

### CODE REVIEW

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:sf

File: modules/apps/portal-language/portal-language-override-test/src/testFunctional/tests/LanguageOverride.testcase [hunk 1]
```diff
@@ -117,6 +117,30 @@ definition {
...
+				value1 = "0-analytics-cloud-connection");
+
+			AssertTextEquals(
+				key_translation = "Analytics Cloud Connection",
+				locator1 = "LanguageOverride#VIEW_PAGE_LANGUAGE_KEY_TRANSLATION",
+				value1 = "Analytics Cloud Connection");
+
+			AssertTextEquals(
+				locator1 = "Search#SEARCH_INFO",
+				value1 = "1 Results for 0-analytics-cloud-connection");
```
Comment (patriciajeanperez) [inline modules/apps/portal-language/portal-language-override-test/src/testFunctional/tests/LanguageOverride.testcase:140]: @melodywu can we update the assertions to assert text based on whatever appears first from top to bottom for readability?

**Review by patriciajeanperez — COMMENTED** [review]
@melodywu Can we also search for a translation that is not on the first page to make it more robust?

**Comment by melodywu** [comment]
> @melodywu Can we also search for a translation that is not on the first page to make it more robust?

Ok

**Comment by melodywu** [comment]
ci:test:bundle

**Pull request closed without merging**